docs/develop/java/core-application.mdx

```diff
@@ -404,47 +404,85 @@ Ensure that your Workflow or Client can handle an Object type return or is able
 
 **How to customize your Activity Type using the Java SDK.**
 
-Activities have a Type that are referred to as the Activity name.
-The following examples demonstrate how to set a custom name for your Activity Type.
+Each Activity has a Type called the Activity 'name'.
+This name appears in the Workflow Execution Event History Summary, listed for each Activity Task.
+The name lets you identify Activity Types called during the Execution.
 
-The Activity Type defaults to method name, with the first letter of the method name capitalized, and can be customized using `namePrefix()` or `{ActivityMethod.name()}` to ensure they are distinct.
+Custom Activity Type names prevent name collisions across interfaces and Workflows.
+They offer descriptive Activity method names without concerns about re-using those names elsewhere in your project.
+They also support code management, especially in larger projects with many Activities.
+For example, you might use a prefix to group related activities together.
+Custom names also distinguish keys for gathering metrics without name conflicts.
```

Comment:
I can't verify the accuracy of this. I'm not saying that it's wrong, just that I don't know enough about metrics to confirm that it's right (and don't have time to research it). I'd recommend either verifying this with an engineer or omitting the statement.

Reply:
Metrics info was pulled right from the code doc for the SDK.